packages/focusvisible/src/useFocusVisible.ts

```diff
@@ -104,19 +114,19 @@ export function useFocusVisible({
      * Add the `:focus-visible` class to the given element if it was not added by
      * the consumer.
      */
-    const addFocusVisibleClass = el => {
+    const addFocusVisibleClass = (el: HTMLElement) => {
       if (isFocused(el)) {
         return;
       }
 
       el.classList.add(className);
-      el.setAttribute(dataAttribute, true);
+      el.setAttribute(dataAttribute, 'true');
```

Comment:
`setAttribute` is [typed](https://github.com/microsoft/TypeScript/blob/master/lib/lib.dom.d.ts#L5332) to take a string as the second argument.